doc/changes/changes49.xml

```diff
@@ -0,0 +1,1196 @@
+one argument, or even an arbitrary number. E.g. <C>{a,b} -> a+b</C>
+is a shorthand for <C>function(a,b) return a+b; end</C>. For details,
+see <Ref BookName="ref" Sect="Function"/>. (For technical details,
+e.g. why we did not choose the syntax <C>(a,b) -> a+b</C>, see
+<URL Text="#490">https://github.com/gap-system/gap/pull/490</URL>).
+</Item>
+<Item>
+Function calls, list accesses and records now can be nested. For example,
+you can now write <C>y := f().x;</C> (essentially equivalent to
+<C>y := f();; y := y.x;</C>), which previously would have resulted in
+an error; see <URL Text="#457">https://github.com/gap-system/gap/issues/457</URL>
+and <URL Text="#462">https://github.com/gap-system/gap/pull/462</URL>).
+</Item>
+<Item>
+The libraries of small, primitive and transitive groups which previously
+were an integral part of &GAP; were split into three separate packages
+<URL Text="PrimgGrp">http://gap-packages.github.io/primgrp/</URL>,
+<URL Text="SmallGrp">https://gap-packages.github.io/smallgrp/</URL> and
+<URL Text="TransGrp">http://www.math.colostate.edu/~hulpke/transgrp/</URL>.
+For backwards compatibility, these are required packages in &GAP; 4.9
+(i.e., &GAP; will not start without them). We plan to change this for
+&GAP; 4.10, once all packages which currently implicitly rely on these
+new packages had time to add explicit dependencies on them
+(<URL Text="#1650">https://github.com/gap-system/gap/pull/1650</URL>,
+<URL Text="#1714">https://github.com/gap-system/gap/pull/1714</URL>).
+</Item>
+<Item>
+The performance of &GAP;'s sorting functions (such as
+<Ref BookName="ref" Oper="Sort"/>, <Ref BookName="ref" Oper="SortParallel"/>,
+etc.) has been substantially improved, in some examples by more than
+a factor of four: as a trivial example, compare the timing for
+<C>Sort([1..100000000] * 0)</C>. As a side effect, the result
+of sorting lists with equal entries may produce
+different answers compared to previous &GAP; versions.
+If you would like to make your code independant of the exact
+employed sorting algorithm, you can use the newly added
+<Ref BookName="ref" Oper="StableSort"/>, <Ref BookName="ref" Oper="StableSortBy"/>
+and <Ref BookName="ref" Oper="StableSortParallel"/>. (For some technical details,
+see <URL Text="#609">https://github.com/gap-system/gap/pull/609</URL>).
+</Item>
+<Item>
+We removed our old home-grown big integer code, and instead always use the
+GMP based big integer code. This means that the GMP library now is a
+required dependency, not just an optional one.
+</Item>
+<Item>
+A number of improvements have been made to <Ref BookName="ref" Oper="Random"/>. 
+These may lead to different sequences of numbers being created.
+On the up side, many more methods for <Ref BookName="ref" Oper="Random"/>
+(and other <C>RandomXYZ</C> operations) now optionally take an explicit
+<Ref BookName="ref" Oper="RandomSource"/> as first argument
+(but not yet all: help with this is welcome). Some relevant pull requests:
```

Comment:
perhaps we should link to https://github.com/gap-system/gap/issues/1098 after "help is welcome"